code/modules/detectivework/microscope/microscope.dm

```diff
@@ -143,3 +165,15 @@
 	icon_state = "microscope"
 	if(sample)
 		icon_state += "slide"
+
+/obj/machinery/microscope/proc/can_analyse_fingerprints()
+	return TRUE
+
+/obj/machinery/microscope/proc/can_analyse_swabs()
+	return TRUE
+
+/obj/machinery/microscope/proc/can_analyse_fibers()
+	return TRUE
+
+/obj/machinery/microscope/proc/can_analyse_cells()
+	return TRUE
```

Comment:
These shouldn't be procs if all they do is act as variables, use a bitfield variable instead